providers/src/airflow/providers/fab/www/static/js/toggle_theme.js

```diff
@@ -0,0 +1,46 @@
+/*!
```

Comment:
Copy of `airflow/www/static/js/toggle_theme.js`

Reply:
I think we can drop support for dark mode. It won't be consistent with AF3 anyways.

Reply:
Done